shared/haml/hamburger.haml

```diff
@@ -40,23 +45,32 @@
 
   show_teacher_options = "hide-always"
   show_student_options = "hide-always"
+  show_user_options = "hide-always"
   show_pegasus_options = "hide-always"
+  show_help_options = "show-always"
+  show_signed_out_options = "hide-always"
 
   hamburger_class = "none"
 
   if level
     # The header is taken over by level-related UI, so we need the hamburger
     # to show whatever would show up in the header at desktop (and mobile) widths.
     if user_type == "teacher"
+      show_user_options = "show-always"
       show_teacher_options = "show-always"
     elsif user_type == "student"
+      show_user_options = "show-always"
       show_student_options = "show-always"
+    else
+      show_user_options = "show-always"
+      show_signed_out_options = "show-always"
     end
 
     # Regardless of user type, if they are English, then they also need the pegasus
     # options in the hamburger.
     if language == "en"
       show_pegasus_options = "show-always"
+      show_help_options = "show-always"
```

Comment:
Alternatively, maybe more consistent to start it as hidden and keep this line here, per comment above.